apps/meteor/server/models/raw/ModerationReports.ts

```diff
@@ -18,12 +19,11 @@ export class ModerationReportsRaw extends BaseRaw<IModerationReport> implements
 
 	modelIndexes(): IndexDescription[] | undefined {
 		return [
-			{ key: { 'ts': 1, 'reports.ts': 1 } },
-			{ key: { 'message.u._id': 1, 'ts': 1 } },
-			{ key: { 'reportedUser._id': 1, 'ts': 1 } },
-			{ key: { 'message.rid': 1, 'ts': 1 } },
+			{ key: { '_hidden': 1, 'ts': 1, 'reports.ts': 1 } },
```

Comment:
if I read the code correctly, `reports.ts` is a field created in a pipeline of an aggregation, right? in this case an index will not be used..

Reply:
Thank you, good catch. Yes, since it is created in the `$group` stage, it is not used.